APMrover2/control_modes.cpp

```diff
@@ -189,6 +189,88 @@ void Rover::read_aux_switch()
             cruise_learn_complete();
         }
         break;
+
+    // arm or disarm the motors
+    case CH7_ARM_DISARM:
+        if (aux_ch7 == AUX_SWITCH_HIGH) {
+            arm_motors(AP_Arming::RUDDER);
```

Comment:
Maybe we should add AP_Arming::SWITCH

Reply:
or not they are both stick so rudder is good

Reply:
yes, I noticed this as well.  AP_Arming::RUDDER should probably be generalised to AP_Arming::TRANSMITTER.  But I figured that was out of scope for this change.